### DISCUSSION TITLE
Version 0.12.2

### DISCUSSION THREAD

**Discussion by IvyZX** [google/flax#5151] [General]
## What's Changed
* [flax:examples:wmt] Small linter fixes. by @copybara-service[bot] in https://github.com/google/flax/pull/5012
* [flax:examples:seq2seq] Create main and default config based on seq2seq.ipynb. by @copybara-service[bot] in https://github.com/google/flax/pull/5119
* [flax:examples:vae] Small linter fixes. by @copybara-service[bot] in https://github.com/google/flax/pull/5014
* [flax:examples:gemma] Fixing linter errors. by @copybara-service[bot] in https://github.com/google/flax/pull/5013
* [flax:examples:sst2] Fix pytype errors. by @copybara-service[bot] in https://github.com/google/flax/pull/5118
* Allow substring matching in `nnx.PathContains` by @thijs-vanweezel in https://github.com/google/flax/pull/5094
* [flax:examples:sst2] Fix notebook error. by @copybara-service[bot] in https://github.com/google/flax/pull/5122
* [flax:examples:ppo] Fix some linter / import issues. #jax-fixit by @copybara-service[bot] in https://github.com/google/flax/pull/5120
* Avoid passing `concrete` argument to `jax.remat` by @copybara-service[bot] in https://github.com/google/flax/pull/5121
* [flax:examples:lm1b_nnx] Update example to work internally. #jax-fixit. by @copybara-service[bot] in https://github.com/google/flax/pull/5125
* [flax:examples:nlp_seq] Create a main.py file to run tests with config files to match other examples. #jax-fixit by @copybara-service[bot] in https://github.com/google/flax/pull/5126
* [jax:benchmarks] Add tracing/lowering benchmarks for a few flax examples. by @copybara-service[bot] in https://github.com/google/flax/pull/4911
* remove abstracted_axes from nnx.jit by @copybara-service[bot] in https://github.com/google/flax/pull/5132
* Pooling operation  by @jorisSchaller in https://github.com/google/flax/pull/5057
* Added is_causal mask argument to flax.nnx.dot_product_attention by @ibbyml in https://github.com/google/flax/pull/5093
* Add out_sharding argument to call methods for layers with jax calls that support it by @samanklesaria in https://github.com/google/flax/pull/5102
* Temporary fix for failing CI by @vfdev-5 in https://github.com/google/flax/pull/5144
* New release 0.12.2 by @IvyZX in https://github.com/google/flax/pull/5149

## New Contributors
* @thijs-vanweezel made their first contribution in https://github.com/google/flax/pull/5094
* @ibbyml made their first contribution in https://github.com/google/flax/pull/5093

**Full Changelog**: https://github.com/google/flax/compare/v0.12.1...v0.12.2

<hr /><em>This discussion was created from the release <a href='https://github.com/google/flax/releases/tag/v0.12.2'>Version 0.12.2</a>.</em>
